FAQ.md

```diff
@@ -0,0 +1,25 @@
+# FAQ
+
+1. **How can I use my local decko instead of the installed `decko` gem so that I can modify the decko?**
+
+   ```
+   $ decko new <Deck Name> --core-dev
+   ```
+
+   This command will then ask for the local decko path. The created deck will be using your local `decko` folder as the gem path in the `GemFile`.
+
+1. **While I started a decko server with the local decko path, it showed the error of `NoMethodError (undefined method 'notable_exception_raised' for #<Card:0x007f8d44fed250>):`.**
+
+   After cloning the decko code from github, there are serval submodules that need to be initialized. You may run the command `git submodule update --init --recursive` in your local decko root directory. Then redo the seeding `bundle exec decko seed`.
+
```

Comment:
I tried to replicate this and couldn't.  So did you have NO code or OLD code in that subrepository?

For now, I'm thinking this should go in a "Troubleshooting" section at the bottom of Contributing.rdoc.  (and we should make sure the README links to the Contributing doc!

Reply:
I do not have any code in the subrepository at the beginning and the error message is not leading to the missing submodule code to a beginner.

Reply:
Yes, I definitely want to make this easy to find.

My main concern is that we separate developer beginners from "normal" beginners.  Normal beginners are installing from gems, so they won't have the submodule problem. Only developers will install the code.

Right now we barely have any documentation for new mod or core developers.  That's a real need!